apis/v1alpha2/converter/httproute.go

```diff
@@ -14,20 +14,20 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package utils
+package converter
```

Comment:
Tiny nit: I'd prefer a name that wasn't so closely linked to conversion. I'm used to that referring to conversion between API versions, and we may want to reserve that terminology in case we ever need a conversion webhook for this project.

Reply:
Gotcha. The original terminology used was "translation" (see https://github.com/kubernetes-sigs/gateway-api/blob/96414f1ce72b2277cde5cb8e8abcd97ea039fe4a/apis/v1alpha2/validation/util/utils.go#L23). 

So, `translator`? `typeconverter`? Other idea?

Reply:
Maybe we should have a `util` subpackage, and then optionally have a `util/translator` subpackage. That would make it easier to add future utils without adding more noise to the API types dir. Don't feel that strongly here, just prefer avoiding "conversion" terminology.

Reply:
Yes, that sounds helpful. To double check:

`apis/vi1alpha2/utils/translator/httproute.go`?

Reply:
LGTM

Reply:
* s/utils/util